miniapp/build.gradle

```diff
@@ -26,6 +26,10 @@ android {
     buildToolsVersion = androidBuildTool
 }
 
+androidExtensions {
+    experimental = true
```

Comment:
why do you need experimental extensions ?

Reply:
it's for using some annotations in Kotlin `@Parcelize`. Extending `Parcelable` requires methods implementation but this annotation implicitly handle it -> short code.